tools/preflight/preflight.go

```diff
@@ -622,6 +618,8 @@ func (o *Run) createDNSPodOnCluster(ctx context.Context, podNameSuffix string, c
 	err = waitUntilPodCondition(ctx, waitOptions)
 	if err != nil {
 		o.Logger.Errorf("DNS pod - %s hasn't reached into ready state", pod.GetName())
+		dnsPodYaml := objToYAML(pod)
+		o.Logger.Warnf("DNS pod created with the following yaml:\n%v", string(dnsPodYaml))
```

Comment:
Here the pod has been created but hasn't reached into the ready state. So, according to the scenario, log an appropriate statement. 
Log statement can be like this: `o.Logger.Warnf("DNS pod failed to reach into ready state. Pod yaml:\n%s", string(objToYAML(pod)))`